app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -40,17 +48,29 @@ class ExplorationActivity : InjectableAppCompatActivity(), StopStatePlayingSessi
     storyId = intent.getStringExtra(EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY)
     explorationId = intent.getStringExtra(EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY)
     backflowScreen = intent.getIntExtra(EXPLORATION_ACTIVITY_BACKFLOW_SCREEN_KEY, -1)
-    explorationActivityPresenter.handleOnCreate(this, internalProfileId, topicId, storyId, explorationId, backflowScreen)
+    explorationActivityPresenter.handleOnCreate(
+      this,
+      internalProfileId,
+      topicId,
+      storyId,
+      explorationId,
+      backflowScreen
+    )
   }
 
   companion object {
     /** Returns a new [Intent] to route to [ExplorationActivity] for a specified exploration. */
 
-    internal const val EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY = "ExplorationActivity.profile_id"
+    internal const val EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY =
```

Comment:
Just to confirm, as per the discussion in the meeting regarding keys, is this suppose to be `EXPLORATION_ACTIVITY_PROFILE_ID_EXTRA_KEY` instead of `EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY`. Ditto elsewhere

Reply:
I updated only ExplorationFragment arguments will update this as mentioned in the Explanation as this was not introduced by me.

Reply:
Okay, got it.